src/assets/scss/govuk-frontend/overrides/_overrides.scss

```diff
@@ -196,4 +196,8 @@ $govuk-radios-focus-width: $govuk-focus-width + 1px;
     outline:3px solid $focus-colour;
 	outline-offset: 0;
 	background-color: $focus-colour;
-}
\ No newline at end of file
+}
+
+#businessName {
+    word-break: break-all;
+}
```

Comment:
I'm not sure the benefit of fixing it here - we'd have to do it on a much wider scale than just a single table cell.

I'd say take this out for now.

Reply:
My concern was that doing it on a wider scale will affect cells (and other usages of cells in tables too potentially) beyond the scope of this ticket. I will push up a version that affects it on a wider scale but we will have to be aware of that.